vNext/channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -501,18 +501,33 @@ export class Sender implements IChannelControlsAI {
     private _beaconSender(payload: string[], isAsync: boolean) {
         var url = this._config.endpointUrl();
         var batch = this._buffer.batchPayloads(payload);
+        var browserLimit = 63000;
+        var count = 0;
+        var overLimit = false;
+
+        for (var i = 0; i < payload.length; i++){
```

Comment:
I'm wondering if instead of having these checks we can instead just fallback to xhr whenever `sendBeacon` fails to queue the event. This way we don't need to do any size checks, etc

Reply:
makes sense